.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy
+
+on:
+  push:
```

Comment:
is `push` the same as `merge` here? i think we don't want to trigger the upload to `s3` until we have approved the PR and merged it?

Reply:
ya correct - the deploy workflow runs only on commits to the main branch. I also made the main branch protected meaning only changes via pull requests can push to it